src/components/Article.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Article.scss';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => (
+  <>
+    <article className="article">
+      <h1 className="article__title">{title}</h1>
```

Comment:
What if we don't get a title? Will we render empty h1? Could we don't render h1 if we don't have title value?